### ISSUE TITLE
Uncaught Error: Cannot get property 'id' on missing remote object 3955

### ISSUE THREAD

**Issue by HyunJu1** [atom/atom#21164]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.42.0 x64
**Electron**: 4.2.7
**OS**: Unknown macOS version
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: Cannot get property 'id' on missing remote object 3955

```
At /Applications/Atom.app/Contents/Resources/electron.asar/renderer/api/remote.js:222

Error: Cannot get property 'id' on missing remote object 3955
    at throwRPCError (/Applications/Atom.app/Contents/Resources/electron.asar/browser/rpc-server.js:148:17)
    at /Applications/Atom.app/Contents/Resources/electron.asar/browser/rpc-server.js:428:5
    at EventEmitter.ipcMain.on.args (/Applications/Atom.app/Contents/Resources/electron.asar/browser/rpc-server.js:273:21)
    at EventEmitter.emit (events.js:182:13)
    at WebContents.<anonymous> (/Applications/Atom.app/Contents/Resources/electron.asar/browser/api/web-contents.js:368:21)
    at WebContents.emit (events.js:182:13)

```

### Commands

```
     -3:11.3.0 application:add-project-folder (input.hidden-input)
  2x -2:04.5.0 core:close (atom-pane.pane.active)
```

### Non-Core Packages

```
atom-beautify 0.33.4 
atom-debug-ui 1.0.4 
atom-ide-ui 0.13.0 
atom-python-run 0.9.7 
busy-signal 2.0.1 
color-picker 2.3.0 
highlight-selected 0.17.0 
ide-java 0.9.0 
ide-php 0.7.18 
ide-python 1.5.0 
intentions 1.1.5 
linter 2.3.1 
linter-php 1.6.1 
linter-ui-default 1.8.1 
minimap 4.29.9 
minimap-highlight-selected 4.6.1 
php-debug 0.3.5 
pigments 0.40.2 
script 3.25.0 
```

**Comment by sadick254** [comment]
Thanks for contributing!

All new issues now require steps to reproduce (also called "repro steps") the problem or the behavior you wish to see in Atom. We do this so we can be certain that we are understanding what it is you're describing and don't prematurely close the issue or implement the wrong feature. If you need help writing detailed repro steps, please see the [How Can I Contribute? section of the Atom CONTRTIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).

**Comment by sadick254** [comment]
Thanks for reporting this!

This has already been fixed on the latest version of atom `v1.51.0`. Consider upgrading to the latest version. Am happy to reopen this if you run into the same issue on the latest version.
